exporter/redis.go

```diff
@@ -133,6 +133,11 @@ func (e *Exporter) initGauges() {
 		Name:      "slave_info",
 		Help:      "Information about the Redis slave",
 	}, []string{"addr", "alias", "master_host", "master_port", "read_only"})
+	e.metrics["start_time_seconds"] = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: e.namespace,
+		Name:      "start_time_seconds",
+		Help:      "Start time of the Redis instance since unix epoch in seconds.",
```

Comment:
"since" implies a duration.  Suggest changing this to: "Start time of the Redis instance in unix epoch seconds."

Reply:
the epoch is always just seconds since 0, or 00:00:00 UTC, Thursday, 1 January 1970, see https://github.com/prometheus/client_golang/blob/master/prometheus/process_collector.go#L82